doc/rados/operations/placement-groups.rst

```diff
@@ -559,100 +559,103 @@ resources.
+Other values will result in an uneven distribution of data across your OSDs. Do
+not increase the number of PGs unless it is possible to change the number of
+PGs to the nearest power of two.
 
-As an example, for a cluster with 200 OSDs and a pool size of 3
-replicas, you would estimate your number of PGs as follows
+For example, if you have a cluster with 200 OSDs and a pool size of 3 replicas,
+estimate the number of PGs as follows:
 
-  :math:`\frac{200 \times 100}{3} = 6667`. Nearest power of 2: 8192
+  :math:`\frac{200 \times 100}{3} = 6667`. Rounded up to the nearest power of 2: 8192.
 
-When using multiple data pools for storing objects, you need to ensure
-that you balance the number of placement groups per pool with the
-number of placement groups per OSD so that you arrive at a reasonable
-total number of placement groups that provides reasonably low variance
-per OSD without taxing system resources or making the peering process
-too slow.
+When using multiple data pools to store objects, make sure that you balance the
+number of PGs per pool against the number of PGs per OSD so that you arrive at
+a reasonable total number of PGs. It is important to find a number that
+provides reasonably low variance per OSD without taxing system resources or
+making the peering process too slow.
 
-For instance a cluster of 10 pools each with 512 placement groups on
-ten OSDs is a total of 5,120 placement groups spread over ten OSDs,
-that is 512 placement groups per OSD. That does not use too many
-resources. However, if 1,000 pools were created with 512 placement
-groups each, the OSDs will handle ~50,000 placement groups each and it
-would require significantly more resources and time for peering.
+For example, suppose you have a cluster of 10 pools, each with 512 PGs on 10
+OSDs. That amounts to 5,120 PGs distributed across 10 OSDs, or 512 PGs per OSD.
+This cluster will not use too many resources. However, in a cluster of 1,000
+pools, each with 512 PGs on 10 OSDs, the OSDs will have to handle ~50,000 PGs
+each. This cluster will require significantly more resources and significantly
+more time for peering.
 
-You may find the `PGCalc`_ tool helpful.
+For determining the optimal number of PGs per OSD, we recommend the `PGCalc`_
+tool.
 
 
 .. _setting the number of placement groups:
 
-Set the Number of Placement Groups
-==================================
+Setting the Number of PGs
+=========================
 
-To set the number of placement groups in a pool, you must specify the
-number of placement groups at the time you create the pool.
-See `Create a Pool`_ for details.  Even after a pool is created you can also change the number of placement groups with:
+Setting the initial number of PGs in a pool must be done at the time you create
+the pool. See `Create a Pool`_ for details. However, even after a pool is
+created, you can change the number of PGs by running a command of the following
+form:
```

Comment:
, if you aren't employing the pg_autoscaler to manage `pg_num` values.

Reply:
Accepted with syntactical modifications made to facilitate the cramming of this suggestion into the Procrustean bed of the extant sentence.